pages/common/ld.md

```diff
@@ -0,0 +1,16 @@
+# ld
+
+> The GNU ELF linker.
+> More information: <https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_mono/ld.html>.
```

Comment:
Why exactly did you choose to use 28 year old documentation? I'd prefer using current one:

```suggestion
> More information: <https://sourceware.org/binutils/docs-2.38/ld.html>.
```